ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-commands.sh

```diff
@@ -14,21 +14,24 @@ source ./venv_qe/bin/activate
 ES_PASSWORD=$(cat "/secret/password")
 ES_USERNAME=$(cat "/secret/username")
 
-
-git clone https://github.com/cloud-bulldozer/e2e-benchmarking --depth=1
+git clone -b "2549_arm-bits" https://github.com/afcollins/e2e-benchmarking --depth=1
 pushd e2e-benchmarking/workloads/kube-burner-ocp-wrapper
+export WORKLOAD=cluster-density-v2
 
 current_worker_count=$(oc get nodes --no-headers -l node-role.kubernetes.io/worker=,node-role.kubernetes.io/infra!=,node-role.kubernetes.io/workload!= --output jsonpath="{.items[?(@.status.conditions[-1].type=='Ready')].status.conditions[-1].type}" | wc -w | xargs)
 
+# Run a warmup for scheduling inconsistencies
+ES_SERVER="none" ITERATIONS=${current_worker_count} CHURN=false ./run.sh
+
+# The measurable run
 iteration_multiplier=$(($ITERATION_MULTIPLIER_ENV))
 export ITERATIONS=$(($iteration_multiplier*$current_worker_count))
-export WORKLOAD=cluster-density-v2
 
 export ES_SERVER="https://$ES_USERNAME:$ES_PASSWORD@search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com"
-
+export EXTRA_FLAGS+=" --gc-metrics=true "
```

Comment:
`+=` here because we want to preserve the default value of --churn-duration=20m. That is why the run yesterday failed.